values.yaml

```diff
@@ -103,3 +103,8 @@ podAnnotations: {}
 # - "-Dlogging.level.io.armory.cloud.wormhole=DEBUG"
 extraOpts: []
 
+# Resource management for the remote-network-agent container
+memoryRequest: "1500Mi"
```

Comment:
defaults set here, but the helper function will also set these values in case this gets removed by the user